programs/psa/crypto_examples.c

```diff
@@ -43,13 +43,13 @@
         }                                                                     \
     } while ( 0 )
 
-#if !defined(MBEDTLS_PSA_CRYPTO_C) || !defined(MBEDTLS_AES_C) || \
+#if !defined(MBEDTLS_PSA_CRYPTO_C) || !defined(PSA_WANT_KEY_TYPE_AES) || \
```

Comment:
We haven't started to update the dependencies in the programs yet AFAIK. Do you need that for a particular reason? If not I would suggest not doing this for the time being.

Reply:
Indeed, I'll revert this change. I can squash if people feel that is more appropriate.